diff --git a/src/data/workshop-strings.ts b/src/data/workshop-strings.ts
index 172e0399..fb6043f0 100644
--- a/src/data/workshop-strings.ts
+++ b/src/data/workshop-strings.ts
@@ -198,6 +199,7 @@ export const STR = {
         title: 'პულტი',
         notes: 'ჩემი ჩანაწერები',
         pinListHeader: 'პასუხები, დააჭირეთ ვარსკვლავს ეკრანზე გამოსატანად',
+        findersHeader: 'ვინ რა მონიშნა',
         unpin: 'დამაგრების მოხსნა',
         openDisplay: 'ეკრანის გახსნა (გასაზიარებლად)',
         stop: 'ვორქშოპის დასრულება',
